src/main/java/offtop/UserService/listeners/UserDataProducer.java

```diff
@@ -0,0 +1,25 @@
+package offtop.UserService.Services;
+
+import java.util.*;
+import org.slf4j.LoggerFactory;
+import org.apache.kafka.clients.producer.Producer;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+import offtop.UserService.Models.User;
+
+@Service
+public class UserDataProducer{
+
+   private static final Logger logger = LoggerFactory.getLogger(ProducerService.class);
```

Comment:
This needs to change to `private static final Logger logger = LoggerFactory.getLogger(UserDataProducer.class);`